lib/perl/OVH/Bastion/log.inc

```diff
@@ -295,11 +296,11 @@ sub _sql_log_insert_file {
         if ($sqltype eq 'local') {
             $prepare =
                 "INSERT INTO connections"
-              . "(uniqid,timestamp,timestampusec,account,cmdtype,allowed,hostfrom,ipfrom,portfrom,bastionip,bastionport,hostto,ipto,portto,user,plugin,params,comment)"
-              . "VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)";
+              . "(uniqid,timestamp,timestampusec,account,cmdtype,allowed,hostfrom,ipfrom,portfrom,bastionip,bastionport,hostto,ipto,portto,user,plugin,params,comment,ttyrecfile)"
+              . "VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)";
             @execute = (
```

Comment:
Note that you might leverage on this array to prevent the `?` thingy above:

```perl
. 'VALUES (' . ('?,' x @execute - 1) . '?)';
```

Reply:
Nice one, committing this on a future misc branch :)